src/scenarios/shared/runner.py

```diff
@@ -533,12 +533,12 @@ def run(self):
                 # Create the fullydrawn command
                 fullyDrawnRetrieveCmd = xharness_adb() + [ 
                     'shell',
-                    f"logcat -d | grep 'ActivityTaskManager: Fully drawn {self.packagename}'"
+                    f"logcat -d | grep -E 'ActivityManager|ActivityTaskManager: Fully drawn {self.packagename}'"
                 ]
 
                 basicStartupRetrieveCmd = xharness_adb() + [ 
                     'shell',
-                    f"logcat -d | grep 'ActivityTaskManager: Displayed {androidHelper.activityname}'"
+                    f"logcat -d | grep -E 'ActivityManager|ActivityTaskManager: Displayed {androidHelper.activityname}'"
```

Comment:
```suggestion
                    f"logcat -d | grep -E 'ActivityManager|ActivityTaskManager' | grep ': Displayed {androidHelper.activityname}'"
```

I realized it might be safer to do the regex grep for "ActivityManager" or "ActivityTaskManager" separately from the `androidHelper.activityname` as it might happen that some character inside the activity name is a special regex char.

Reply:
I think this second update looks good 👍.